### DISCUSSION TITLE
[Question] Mapping keymaps to toggle comments

### DISCUSSION THREAD

**Discussion by liketoeatcheese** [numToStr/Comment.nvim#336] [Q&A] [answered]
Hey mate,

First of all, awesome work on the plugin!

To my question, to comment on the whole paragraph, I normally have to vip to select the whole paragraph and then gc to toggle the comment. I want to shorten this by creating a keymap with gcp:
keymap("n", "gcp", "vipgc", opts)

But when I do that, it only selects the paragraph without commenting it, is there a way to call that as a lua function? Something like this:
keymap("n", "gcp", "vip:CommentToggle", opts)

I hope that makes sense

**Comment by numToStr** [comment, marked as answer]
You can just do `gcip` to comment out the paragraph.

**Reply by liketoeatcheese** [reply]
That works. Thanks heaps
